docs/feature_request.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+
+title: "REQUEST: "
```

Comment:
As above, when the correct label is applied, I believe prefixing "REQUEST:" to the title adds unneeded distraction.